client/src/components/LoginForm/LoginForm.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```

Comment:
As far as I know you don't need to import React here.

Reply:
Okay, I will try that out. I always used the 'template'-code from storybook